### ISSUE TITLE
NER : why is NER and NEL not fusioned? 

### ISSUE THREAD

**Issue by flavienbwk** [UniversalDataTool/universal-data-tool#315]
Hi,

When I annotate sentences, why do we need to click on the switch button to go to linking mode? Couldn't you fusion these two functionalities?

It is really not convinient having to click on each option every time. 

Thanks

**Comment by seveibar** [comment]
I agree, it would also be more convenient to have a hotkey for the switching if the switching were to stay. Thanks for reporting!

**Comment by seveibar** [comment]
The fix for this will be in the [react-nlp-annotate](https://github.com/UniversalDataTool/react-nlp-annotate) repository
